src/main/kotlin/com/demonwav/mcdev/platform/canary/CanaryModule.kt

```diff
@@ -50,6 +52,15 @@ class CanaryModule<out T : AbstractModuleType<*>>(facet: MinecraftFacet, overrid
         return canaryInf
     }
 
+    fun getNeptuneInf(): VirtualFile? {
```

Comment:
You can implement this as regular property (using a custom getter and a private setter)